contracts/prize-pool/PrizePool.sol

```diff
@@ -23,14 +25,12 @@ abstract contract PrizePool is Initializable, AbstractYieldService, BaseRelayRec
   event CapturedAward(uint256 amount);
   event Deposited(address indexed operator, address indexed to, address indexed token, uint256 amount);
   event Awarded(address indexed winner, address indexed token, uint256 amount);
+  event AwardedExternal(address indexed winner, address indexed token, uint256 amount);
   event InstantWithdrawal(address indexed operator, address indexed from, address indexed token, uint256 amount, uint256 exitFee, uint256 sponsoredExitFee);
   event TimelockedWithdrawal(address indexed operator, address indexed from, address indexed token, uint256 amount, uint256 unlockTimestamp);
   event TimelockedWithdrawalSwept(address indexed operator, address indexed from, uint256 amount);
 
-  address internal constant SENTINAL_TOKEN = address(0x1);
-
-  mapping(address => address) internal _tokens;
-  uint256 tokenCount;
+  MappedSinglyLinkedList.Mapping _tokens;
   ComptrollerInterface public comptroller;
   ControlledToken public timelock;
```

Comment:
"timelock" token variable is no longer used

Reply:
Let's ✂️